elasticsearch5/src/main/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndex.java

```diff
@@ -368,6 +365,75 @@ private void addElementWithScript(Graph graph, Element element, Authorizations a
         }
     }
 
+    @Override
+    public <TElement extends Element> void updateElement(Graph graph, ExistingElementMutation<TElement> mutation, Authorizations authorizations) {
```

Comment:
To make this method a little more readable/shorter can you split it up into 3-4 steps perhaps

Reply:
@joeferner Good idea. Done with https://github.com/visallo/vertexium/pull/243/commits/6359cf5913d91630bebb0121a9dcb9b3ee65b8e3